packages/components/psammead-brand/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 7.3.13 | [PR#4586](https://github.com/bbc/psammead/pull/4586) Fix a11y spans and comma bugs |
```

Comment:
Should we be more specific? Fix for TalkBack incorrectly reading links in nested spans? And does this PR fix the comma bug?

Reply:
No it does not fix the comma bug, I need to update these PRs 😅

Reply:
And yes, will try to be more specific